controls/gridview/getting-started.md

```diff
@@ -13,78 +13,101 @@ previous_url: gridview-getting-started
 
 | RELATED VIDEOS |  
 | ------ |
-|[RadGridView for WinForms Webinar](http://tv.telerik.com/watch/winforms/webinar/radgridview-for-winforms-webinar)<br>Lino Tadros from Falalel provides an in-depth overview of Telerik's RadGridView for WinForms (Runtime: 01:45:11)|
+|[RadGridView for WinForms Webinar](https://www.telerik.com/videos/winforms/radgridview-for-winforms-webinar)<br>Lino Tadros from Falalel provides an in-depth overview of Telerik's RadGridView for WinForms (Runtime: 01:45:11)|
 
-The example below demonstrates creating a hierarchical grid presenting two tables *Product Category* and *Product Information*.
+The following tutorial demonstrates how to create a hierarchical grid that presents two tables: *Product Category* and *Product Information*.
 
->caption Figure 1: The final result from the tutorial.
+>caption Figure 1: The final result
 
 ![grid-getting-started 001](images/grid-getting-started001.png)
 
-## Setting up the Form 
+## Setting up the Form
+
+1. Create a new __Windows Forms Application__ project.
 
-1. Create a new __Windows Application__ project.
-            
 1. Drop a __RadGridView__ control on the form.
 
-1. Set the RadGridView.__Dock__ property to *Fill* in the Properties window of Visual Studio.
```

Comment:
Is it the properties window of VS, or is it the properties window for the control? I could be wrong, but I believe this window is for the control.